<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCAccess2Records</class>
 <widget class="QFrame" name="NewCAccess2Records">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>出入记录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(D:/WinParkUI/debug/Image/NewIcon/CommonBigBG-normal1366x768.jpg);</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>1231</width>
     <height>731</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <widget class="QTableWidget" name="tableAccessRecord">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1221</width>
      <height>201</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>单击【卡号列】查看进出图片。
双击单元格查看库中库记录。</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>卡号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>卡类型</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>应收费</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>实收费</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>优惠原因</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>卡片自编号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>进入通道</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>进入时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>离开通道</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>离开时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>收费车型</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>进入车牌</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>离开车牌</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>记录ID</string>
     </property>
    </column>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>1221</width>
      <height>321</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1421</width>
      <height>321</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>车辆进入图像</string>
     </attribute>
     <widget class="QGroupBox" name="gb0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>271</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <property name="title">
       <string>图像1</string>
      </property>
      <widget class="QLabel" name="lblImg0">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>251</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gb1">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>271</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <property name="title">
       <string>图像2</string>
      </property>
      <widget class="QLabel" name="lblImg1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>251</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gb2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>271</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <property name="title">
       <string>图像3</string>
      </property>
      <widget class="QLabel" name="lblImg2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>251</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gb3">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>10</y>
        <width>271</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <property name="title">
       <string>图像4</string>
      </property>
      <widget class="QLabel" name="lblImg3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>251</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>车辆离开图像</string>
     </attribute>
     <widget class="QGroupBox" name="gb5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>271</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>图像2</string>
      </property>
      <widget class="QLabel" name="lblImg5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>251</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gb4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>271</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>图像1</string>
      </property>
      <widget class="QLabel" name="lblImg4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>251</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gb6">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>271</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>图像3</string>
      </property>
      <widget class="QLabel" name="lblImg6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>251</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gb7">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>10</y>
        <width>271</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>图像4</string>
      </property>
      <widget class="QGroupBox" name="groupBox_11">
       <property name="geometry">
        <rect>
         <x>-440</x>
         <y>0</y>
         <width>131</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string>图像1</string>
       </property>
       <widget class="QGraphicsView" name="graphicsView_11">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>111</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_12">
       <property name="geometry">
        <rect>
         <x>-290</x>
         <y>0</y>
         <width>131</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string>图像2</string>
       </property>
       <widget class="QGraphicsView" name="graphicsView_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>111</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_13">
       <property name="geometry">
        <rect>
         <x>-140</x>
         <y>0</y>
         <width>131</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string>图像3</string>
       </property>
       <widget class="QGraphicsView" name="graphicsView_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>111</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="lblImg7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>251</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnMinimalize">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>585</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>7</y>
       <width>1166</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>卡号</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb0">
        <property name="statusTip">
         <string>cardno like '%%1%'</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>卡自编号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="statusTip">
         <string>cardselfno like '%%1%'</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>进入通道</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb3">
        <property name="statusTip">
         <string>inshebeiname like '%%1%'</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>离开通道</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb4">
        <property name="statusTip">
         <string>outshebeiname like '%%1%'</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>进入时间</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb5">
        <property name="statusTip">
         <string>intime like '%%1%'</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>离开时间</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb6">
        <property name="statusTip">
         <string>outtime like '%%1%'</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>进入车牌</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb7">
        <property name="statusTip">
         <string>carcp like '%%1%'</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>离开车牌</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb8">
        <property name="statusTip">
         <string>carcpout like '%%1%'</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnQuery">
        <property name="toolTip">
         <string>全【无】查询所有记录。</string>
        </property>
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSerach">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>查找</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>5</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);
color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblClose">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>8</y>
     <width>30</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cb1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>cardkind like '%%1%'</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>40</y>
     <width>52</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>卡类型</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
